editors/vscode/client/package-lock.json

```diff
@@ -0,0 +1,902 @@
+{
+	"name": "swarm-lsp-client",
```

Comment:
I read the arguments for why package-lock files should be committed, but it seems like nonsense to me:
- https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

I think it boils down to "lots of packages break or make breaking changes" even with semantic versioning. :shrug:

Hopefully, it will not bother us too much as we do not change VS Code plugin too often. :slightly_smiling_face: